doc/api/util.md

```diff
@@ -1631,6 +1631,71 @@ const module = new WebAssembly.Module(wasmBuffer);
 util.types.isWebAssemblyCompiledModule(module);  // Returns true
 ```
 
+### util.createPromiseHook(hook)
+<!-- YAML
+added: REPLACEME
+-->
+
+* `hook` {Function}
+  * `type` {string}
+    * `resolve` Called when a promise resolve function is called. The promise
+      is not fulfilled or rejected at this point.
+    * `rejectWithNoHandler` Called when a promise rejects without any handler.
+    * `handlerAddedAfterReject` Called when a promise handler is added to a
+      promise that rejected without a handler.
+    * `rejectAfterResolve` Called when a promise reject function is called
+      after resolution.
+    * `resolveAfterResolve` Called when a promise resolve function is called
```

Comment:
The actual value in code is `resolveAfterResolved` (note missing 'd'). Would 'Settled' be better here?

Reply:
whoops will fix. also settled and resolved are not equivalent.